src/components/InputField/InputField.module.css

```diff
@@ -51,6 +51,20 @@
   color: var(--dark-green);
 }
 
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus {
+  -webkit-box-shadow: 0 0 0px 1000px var(--dark-gray) inset;
+  -webkit-text-fill-color: inherit;
+  background-color: var(--dark-gray);
+}
+
+.readOnly {
+  background-color: #b2b3b7 !important;
```

Comment:
Using !important is most of the time not a good practice.
Is there another way in which this could be implemented?